1d-burgers/inf_cont_burgers.py

```diff
@@ -106,16 +110,19 @@ def predict(self, X_star):
     X_u_train, u_train, X_f, ub, lb = prep_data(
         path, hp["N_u"], hp["N_f"], noise=0.0)
 
+# [mtsokol] for bigger values slows down drastically
+X_star = X_star[::100, :]
+u_star = u_star[::100, :]
```

Comment:
What's going on here?

Reply:
With full `X_star` it takes a lot of time to compute final error so I taken every hundredth to make it faster.